paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/viewmodel/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -273,19 +277,50 @@ class PaymentSheetPlaygroundViewModel(
         returnUrl: String,
         backendUrl: String,
     ): CreateIntentResult {
-        // Note: This is not how you'd do this in a real application. Instead, your app would
-        // call your backend and create (and optionally confirm) a payment or setup intent.
-        return CreateIntentResult.Success(clientSecret = clientSecret.value!!)
+        val initializationType = initializationType.value
+
+        val clientSecret = if (initializationType == InitializationType.DeferredMultiprocessor) {
+            PaymentSheet.IntentConfiguration.DISMISS_WITH_SUCCESS
+        } else {
+            // Note: This is not how you'd do this in a real application. Instead, your app would
```

Comment:
Should we update our backend and sample to do this too then?

Reply:
I don’t think that’s necessary for the playground. We do things properly in the example screens.